README.md

```diff
@@ -9,16 +9,18 @@ Features
 Examples
 --------
 
+
 Installation
 ------------
 
-Install spaghetti by running:
+Install the most current version of spaghetti by running:
 
 ```
-$ pip install spaghetti
+$ pip install git+https://github.com/pysal/spaghetti
```

Comment:
You can still prepare  a release for pip as I think that would be useful for users looking to work with spaghetti rather than pysal meta. Going through the packaging of a release for pip would ensure a good user experience. Pulling directly from git might not.